packages/metadata/src/providers/githubMetadata.provider.ts

```diff
@@ -31,43 +35,61 @@ export class GithubMetadataProvider implements IMetadataProvider {
     }
 
     async getChainsMetadata(): Promise<ZKChainMetadata> {
-        const { data } = await this.axios.get(this.chainJsonUrl).catch((e) => {
-            this.logger.error(
-                `Failed to fetch chains metadata from ${this.chainJsonUrl}: ${e.message}`,
-            );
-            throw new FetchError(`Failed to fetch chains metadata: ${e.message}`);
-        });
+        let cachedData = await this.cache.get<ZKChainMetadata | undefined>(
+            `${GITHUB_METADATA_PREFIX}-chains`,
```

Comment:
should we use some sort of unique id here ? like UUID?

Reply:
any suggestions here ? @0xyaco

Reply:
I guess there would be a possibility of, given a shared `Cache` instance between different classes, to overwrite if by accident someone uses the same key in multiple classes.

I can't come up with anything simple as a workaround to that issue right now lol so I think we can keep it as is.

Reply:
an alternative im thinking now would be a wrapper around the cache, like BrandedCache or smth like that:
```
BrandedCache is Cache {
constructor(cache: Cache, prefix: string)
...wraps calls
}
```
and that this class wraps the cache methods and automatically adds the prefix to the key

but seems too much at this point

Reply:
I agree that it's too much, let's just be aware of this issue when defining cache keys.

Reply:
lets keep it as it is nigiri, no need for uuid